src/RpcServer/RpcServer.Blockchain.cs

```diff
@@ -81,7 +80,7 @@ protected virtual JObject GetBlockHeader(JArray _params)
         {
             JObject key = _params[0];
             bool verbose = _params.Count >= 2 && _params[1].AsBoolean();
-            using var snapshot = Blockchain.Singleton.GetSnapshot();
+            var snapshot = system.StoreView;
```

Comment:
Use `Snapshot`?

Reply:
It's okay to use `StoreView` here because headers won't be removed from the ledger.